src/format-phone-number/index.ts

```diff
@@ -17,5 +17,12 @@ import { FormatPhoneNumberFn } from './types';
  * Output: ''
  */
 export const formatPhoneNumber: FormatPhoneNumberFn = (str) => {
-  throw new Error('Not Implemented');
+  const digitsOnly = str.replace(/\D/g, '');
+
+  if (digitsOnly.length !== 12) {
+    return '';
+  } else {
+    const formattedNumber = `+${digitsOnly.slice(0, 2)} (${digitsOnly.slice(2, 5)}) ${digitsOnly.slice(5, 8)}-${digitsOnly.slice(8, 10)}-${digitsOnly.slice(10)}`;
+    return formattedNumber;
+  }
```

Comment:
You have `return` in `if`.
`else` keyword makes no sense, you will not get there, if you entered `if`